litellm/utils.py

```diff
@@ -8602,6 +8602,9 @@ def handle_bedrock_stream(self, chunk):
             # cohere mapping
             elif "text" in chunk_data:
                 text = chunk_data["text"]  # bedrock.cohere
+            # mistral mapping
+            elif "outputs" in chunk_data:
+                text = chunk_data['outputs'][0]['text']
```

Comment:
In my experience, at this point `chunk_data` looks something like this: `{"outputs":[{"text":"The","stop_reason":null}]}`. Therefore, I have two comments:

`stop_reason` should be saved as `finish_reason = chunk_data['outputs'][0]['stop_reason']`

Reply:
hey @GlavitsBalazs thanks for your review here. Curious, how do you use litellm? 

Context - trying to improve for our users